golang/cosmos/x/lien/spec/README.md

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
I find drive-bys acceptable if it would be onerous to put them in a separate PR.

Do you want me not to make this change?  I will remove it in that case.